composer.json

```diff
@@ -28,5 +28,29 @@
         "psr-4": {
             "CakePHP\\": "CakePHP"
         }
+    },
+    "scripts": {
+        "increaseSeverity": "sed -i '' 's/<severity>0<\\/severity>/<!--<severity>0<\\/severity>-->/' CakePHP/ruleset.xml",
+        "includeTestsDirectory": "sed -i '' 's/<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>/<!--<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>-->/' CakePHP/ruleset.xml",
+        "resetRuleset": [
+            "sed -i '' 's/<!--<severity>0<\\/severity>-->/<severity>0<\\/severity>/' CakePHP/ruleset.xml",
+            "sed -i '' 's/<!--<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>-->/<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>/' CakePHP/ruleset.xml"
+        ],
+        "addStandard" : "phpcs --config-set installed_paths $(pwd)",
+        "test": [
+          "@increaseSeverity",
+          "phpunit",
+          "@resetRuleset"
+        ],
+        "cs-check": [
+            "@addStandard",
```

Comment:
I'might not sure for the standard size it is already installed on my machine. Redefiningredients it doesn't hurt I think. Regarding the ruleset it was jut that for this report, given how the tests are written I wanted to stick with all the rules or the CS check but I can remove it (will need to be updated in the Travis config)